asv/util.py

```diff
@@ -392,7 +394,7 @@ def watcher_run():
                 time.sleep(0.1)
                 if time.time() - start_time[0] > timeout:
                     was_timeout[0] = True
-                    proc.terminate()
+                    proc.send_signal(signal.CTRL_BREAK_EVENT)
```

Comment:
Can processes ignore that signal? If yes, is there a way to force termination (like sigkill on posix)?